static/app/views/dashboardsV2/widgetBuilder/header.tsx

```diff
@@ -54,6 +54,7 @@ export function Header({
             value={title}
             onChange={onChangeTitle}
             errorMessage={t('Please set a title for this widget')}
+            maxLength={255}
```

Comment:
We might need to add something similar to this in our builder

![Screen Shot 2022-02-23 at 6 39 58 PM](https://user-images.githubusercontent.com/22846452/155430403-ca5c472c-8f8c-4bc8-9775-25fc55708226.png)

Reply:
sure... we should implement this in the editableText component , but can be done in a separate PR